docs/reference/function-configuration/batching.md

```diff
@@ -0,0 +1,53 @@
+# Event batching
+
+(!) Feature is supported only for:
+* runtimes: [`python`]
+* trigger kinds: [`http`]
+
+The event batching feature allows passing a batch of events instead of a single event to a function handler. 
```

Comment:
The event batching feature allows passing a batch of events, instead of a single event, to a function handler.